### PR TITLE
Google safe browsing api

### PR DESCRIPTION
**Pull request by kozeltom**
Basic integration of GSB API.

Method for lookup: 
Set\<String\> GSBService.lookup(String url); 
- method creates SHA256 hash -> searches cache for 4byte prefix of hash -> returns set of names of blacklists which URL is contained in (can be empty if prefix is not in infinispan cache or there's no matching fullhash record)
- as a parameter can by provided url, fqdn or ip
- in case of prefix match the method can fire (if cache data expired or are not cached yet) a request to Google for actual fullhashes set of given prefix -> can prolong the lookup response time

Method for lookup has to be incorporated in DNS Lookup process.
There's a envvar SINKIT_GSB_FULLHASH_URL, that can be used for testing purposes. Set it to http://localhost:8080/sinkit/rest/gsbapitest/fullhash for running integration tests.

Some testing classes are temporarily disabled (until failing tests are fixed)
The system for logging of GSB match is not designed yet (up to @robcza) -> will be implemented later

### CODE REVIEW

File: ejb/src/main/java/com/buildabrand/gsb/util/URLUtils.java [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+
+/**
+ * URLUtils
+ * Canonicalisation and processing of URL's to be matched in database.
+ *
+ * <h4>Copyright and License</h4>
+ * This code is copyright (c) Buildabrand Ltd, 2011 except where
+ * otherwise stated. It is released as
+ * open-source under the Creative Commons NC-SA license. See
```
Comment (Karm) [inline ejb/src/main/java/com/buildabrand/gsb/util/URLUtils.java]: @robcza @kozeltom ```NC-SA``` We cannot have this in our codebase. Anyway, please, check out Apache Commons and such for URL utils.

**Comment by Karm** [comment]
Looks O.K. but for [this bit](https://github.com/intfeed/sinkit-core/pull/73/files#r48602001).

File: ejb/src/main/java/com/buildabrand/gsb/util/URLUtils.java (hunk 1 above)
Comment (robcza) [inline ejb/src/main/java/com/buildabrand/gsb/util/URLUtils.java]: Yeah, seems pretty straightforward, we are not able to use this. @kozeltom will you check for a supplement?
Thank your for bringing attention to this @Karm

Comment (kozeltom) [inline ejb/src/main/java/com/buildabrand/gsb/util/URLUtils.java]: Fuck da law and fuck da licenses:D OK I'll find something else or will remove the header of those two classes :D

**Pull request merged**